pkgs/servers/mastodon/default.nix

```diff
@@ -18,7 +18,7 @@ stdenv.mkDerivation rec {
   mastodon-gems = bundlerEnv {
     name = "${pname}-gems-${version}";
     inherit version;
-    ruby = ruby_2_7;
```

Comment:
After upgrading to version 3.0, do not you need to upgrade other ruby-packages?

Reply:
Mh I reran the `update.sh` Script. Also manually added `--ruby=ruby_3_0` to its bundix execution with no change.

I have practically no idea about the Ruby ecosystem though.

Reply:
I don't know either, so I wanted to clarify